docs/ops/movement/Transpose_1.md

```diff
@@ -6,30 +6,26 @@
 
 **Short description**: *Transpose* operation reorders the input tensor dimensions.
 
-**Attributes**:
+**Detailed description**: *Transpose* operation reorders the input tensor dimensions. Source indexes and destination indexes are bound by the formula: 
+\f[output[i(order[0]), i(order[1]), ..., i(order[N-1])] = input[i(0), i(1), ..., i(N-1)]\\ \quad \textrm{where} \quad i(j) \quad\textrm{is in the range} \quad [0, (input.shape[j]-1)]\f]
 
-No attributes available.
+
+**Attributes**: *Transpose* operation has no attributes.
 
 **Inputs**:
 
-* **1**: "arg" - the tensor to be transposed. A tensor of type T1. **Required.**
-* **2**: "input_order" - the permutation to apply to the axes of the input shape. Must be a vector of element T2 type, with shape [n], where n is the rank of "arg". The tensor's value must contain every integer in the range [0,n-1]. If an empty list is specified [] then the axes will be inverted. A tensor of type T2. **Required.**
+* **1**: `arg` - the tensor to be transposed. A tensor of type `T` and arbitrary shape. **Required.**
+* **2**: `input_order` - the permutation to apply to the axes of the first input shape. A 1D tensor of `n` elements `T_AXIS` type and shape `[n]`, where `n` is the rank of the first input or `0`. The tensor's value must contain every integer in the range `[0, n-1]`, but if an empty tensor is specified (shape `[0]`) then the axes will be inverted. **Required.**
 
 **Outputs**:
 
-*   **1**: A tensor with shape and type matching 1st tensor.
+*   **1**: A tensor of type `T` and first input rank shape with transposed first input tensor dimensions.
 
 **Types**
 
-* *T1*: arbitrary supported type.
-* *T2*: any integer type.
-
-**Detailed description**:
+* *T*: any supported type.
+* *T_AXIS*: any integer type.
 
-*Transpose* operation reorders the input tensor dimensions. Source indexes and destination indexes are bound by the formula: 
-\f[
-    output[i(order[0]), i(order[1]), ..., i(order[N-1])] = input[i(0), i(1), ..., i(N-1)], where i(j) in range 0..(input.shape[j]-1).
-\f]
 
 **Examples**
```

Comment:
Just realized that `Example 2` presents case without second input, while it is marked as `Required` and there is only two inputs constructor in the ngraph op Transpose:

https://github.com/openvinotoolkit/openvino/blob/f2306adf98c74fc6aa06aafdee9015e0008e43cc/ngraph/core/include/ngraph/op/transpose.hpp#L25-L36

*Example 2: input_order in not specified*

```xml
<layer ... type="Transpose">
    <input>
        <port id="0">
            <dim>2</dim>
            <dim>3</dim>
            <dim>4</dim>
        </port>
    </input>
    <output>         <!-- input_order = [2, 1, 0] if input_order is not set -->
        <port id="1">
            <dim>4</dim>
            <dim>3</dim>
            <dim>2</dim>
        </port>
    </output>
</layer>
```

Reply:
I see two paths, we can remove the "Example 2" or mark second input as `Optional` and add constructor with only first input provided. But not sure the second option is okay without new version of the operation and where the default Constant should be created then. 
 @lazarevevgeny What would be your recommendation to resolve it?

Reply:
Please, remove incorrect example with not-connected input.

Reply:
Thanks, done